core/src/main/scala/pureconfig/package.scala

```diff
@@ -265,27 +265,28 @@ package object pureconfig {
   }
 
   /**
-   * Loads `files` in order, allowing values in later files to backstop missing values from prior, and converts them into a `Config`.
+   * Loads `files` in order, allowing values in later files to backstop missing values from prior, and converts them
+   * into a `Config`.
    *
    * This is a convenience method which enables having default configuration which backstops local configuration.
    *
-   * Note: If an element of `files` references a file which doesn't exist or can't be read, it will silently be ignored.
+   * The behavior of the method if an element of `files` references a file which doesn't exist or can't be read is
+   * defined by the `failOnReadError` flag. With `failOnReadError = false`, such files will silently be ignored while
+   * otherwise they would yield a failure (a `Left` value).
    *
    * @param files Files ordered in decreasing priority containing part or all of a `Config`. Must not be empty.
```

Comment:
Since we now deal property with empty lists of files, I think we should remove the `Must not be empty` sentence from the parameter description.

Reply:
👍 6b8337e

Reply:
Good catch, @jcazevedo!